extensions/remove-paywall/src/remove-paywall.tsx

```diff
@@ -18,6 +18,10 @@ export default async (props: LaunchProps<{ arguments: Arguments.RemovePaywall }>
       throw url;
     }
 
+    if (service === "https//freedium.cfd") {
+      open(service.concat(`/${url}`));
+    }
+
```

Comment:
Hey @artivilla,

First of all, thank you for your contribution. This is not required as the matching webpage is already open in line 26 for the chosen service.
```suggestion
```

Reply:
oh right. for some reason I wrongly assumed at the time of writing it had its own format.